cipher/a1z26/a1z26.go

```diff
@@ -0,0 +1,76 @@
+// a1z26.go
+// description: Implementation of the a1z26 cipher
+// details:
+// The A1Z26 cipher is a simple substitution cipher where each letter is
+// replaced by the number of the order they're in. For example, A corresponds to
+// 1, B = 2, C = 3, etc..
+//
+// author(s): [Focusucof](https://github.com/Focusucof)
+// see a1z26_test.go for tests
+
+// ref: https://www.dcode.fr/letter-number-cipher
+package a1z26
```

Comment:
I don't like the name of the package. I think you can just call it `sub` instead of `a1z26`. As its a simple substitution cipher (though I feel its more like encoding rather than a cipher).